tests/e2e/specs/login_spec.js

```diff
@@ -6,6 +12,21 @@ describe('Login', () => {
     password: 'p@ssword',
   };
 
+  const newCustomer = {
+    firstName: 'Andy',
+    lastName: 'Garcia',
+    email: 'f4831546-0062-470d-b6a9-cacb8e0a2aa4@mailslurp.com',
+    password: 'p@ssword',
+  };
+
+  const newPassword = 'newp@ssword';
+
+  before(() => {
+    cy.wrap(mailslurp.getAllEmails().then(emails => emails.content.forEach((e) => {
+      mailslurp.deleteEmail(e.id);
```

Comment:
[deleteEmail](https://www.mailslurp.com/js/classes/mailslurp/#deleteemail) returns a promise but you are not doing anything with it. Should this not be:

`cy.wrap(mailslurp.getAllEmails().then(emails => Promise.all(emails.content.map((e) => mailslurp.deleteEmail(e.id))))`